karma.conf.js

```diff
@@ -1,3 +1,6 @@
+const path = require('path');
+const testMiddleware = require('./test/middleware');
```

Comment:
this sneakily mounts stub endpoints on the server that hosts the test classes used in browser testing. that way integration tests work both in and out of browser environment